recipes/create-dmg/all/conanfile.py

```diff
@@ -13,24 +13,32 @@ class CreateDmgConan(ConanFile):
     homepage = "https://github.com/create-dmg/create-dmg"
     url = "https://github.com/conan-io/conan-center-index"
     settings = "os", "arch", "compiler", "build_type"
-    no_copy_source = True
 
     @property
     def _source_subfolder(self):
         return "source_subfolder"
 
+    def export_sources(self):
```

Comment:
revert. It's a tool package. https://docs.conan.io/en/latest/creating_packages/existing_binaries.html#downloading-and-packaging-pre-built-binaries

Reply:
I did this in order to be able to patch `create-dmg` to properly access to the `support` directory: https://github.com/conan-io/conan-center-index/pull/11569/files#diff-70f9e2e6516706b1b866dc40b6094b688fa797d9d698871b83399d82e54e371e

Reply:
@uilianries How can I properly patch a tool package ?

Reply:
Follow the pattern that is showed in the link. Any patch should be applied on build step as always.